v4/source/users.yaml

```diff
@@ -1153,9 +1153,14 @@
               type: object
               required:
                 - id
+                - create_at
```

Comment:
You can update the createat field of a user? I'm wondering if that's actually a bug. Seems like something that shouldn't be updatable.

Reply:
Yeah it's a bit weird, the `create_at` field is required for the `UpdateUser` request, but you can't actually modify the field. So it's more of an issue with an unnecessary error being thrown when missing the `create_at` field rather than it being modifiable.

Reply:
I think that would be good to fix; sounds like we have a validation bug.

Reply:
Created an issue for this here [MM-41041](https://mattermost.atlassian.net/browse/MM-41041).